Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM httpd:latest
+
+# Copy configuration file for apache
+COPY ./apache-httpd.conf /usr/local/apache2/conf/httpd.conf
+
+# Expose apache
+EXPOSE 80
+
+# Copy the project files into place.
+COPY public /usr/local/apache2/htdocs/public
```

Comment:
We use react router. It works through redirecting all 404s to index.html. The Rest is resolved through Javascript on the Client.